src/dmd/cli.d

```diff
@@ -715,22 +715,22 @@ struct CLIUsage
     {
         enum maxFlagLength = 18;
         enum s = () {
-            string buf;
+            char[] buf;
             foreach (option; Usage.options)
             {
                 if (option.os.isCurrentTargetOS)
                 {
-                    buf ~= "  -";
-                    buf ~= option.flag;
+                    buf ~= "  -" ~ option.flag;
                     // create new lines if the flag name is too long
                     if (option.flag.length >= 17)
                     {
                             buf ~= "\n                    ";
                     }
                     else if (option.flag.length <= maxFlagLength)
                     {
-                        foreach (i; 0 .. maxFlagLength - option.flag.length - 1)
-                            buf ~= " ";
+                        const n_spaces = maxFlagLength - option.flag.length - 1;
```

Comment:
`n_spaces` is not following D naming conventions.

Reply:
Fixed.